Sec Primaries | Stuff

ProbablyCarl opened the pull request:
- - -
Balance:
 - Security Officers, not Cadets, given access to a 'primary' gun box. This gunbox contains three choices, giving the selection of a 'new' SMG, the beanbag shotgun and the taser carbine.
 - Provides both SMGs, old and new, with automatic fire. This applies to the Sword and Pariah.
 - 'New' SMG given a restriction to a new cartridge type, but has a faster fire rate to compensate.
 - Newer cartridge is a much weaker version of the rifle frangibles, intended for disabling people in the same manner birdshot does. Will absolutely kill someone without armour, but that's not too heavily a concern.
 - Guarantees shrapnel on birdshot.
- - -

ProbablyCarl committed:
Sec Primaries | Stuff
- - -
This is just a temp PR to work out some stupid stuff. Might change and remain if it actually happens to be what we're wanting. Ignore this otherwise instead of freaking out.
- - -
Balance:
 - Security Officers, not Cadets, given access to a 'primary' gun box. This gunbox contains three choices, giving the selection of a 'new' SMG, the beanbag shotgun and the taser carbine.
 - SMG given a proper full auto function, so hopefully it's not absolute garbage still.
 - Frangibles given to SMGs, allowing them to compete with rifles as an alternative.
- - -
Corrections:
 - DarkRP Baton should work again. Hopefully. Maybe. Might cause CI to throw a fit.
- - -

ProbablyCarl committed:
aaaaaaaaaaaaaaaaaaaa

ProbablyCarl committed:
comments out darkrp baton

ProbablyCarl committed:
Merge branch 'dev' into dev

anconfuzedrock commented:
![image](https://user-images.githubusercontent.com/18406892/156702896-69e1a6db-7675-4eb2-a60b-938c0e282380.png)

ProbablyCarl committed:
Introduces new cartridge type, specifically for the 'new' SMG.
Restricts older SMG to lower rate of fire, though retains new automatic fire.
'New' SMG given a restriction to the new cartridge type, but has a faster fire rate to compensate.
Guarantees shrapnel on birdshot.

ProbablyCarl committed:
Merge branch 'dev' of https://github.com/BoHBranch/BoH-Bay into dev

ProbablyCarl committed:
Merge branch 'BoHBranch:dev' into dev

ProbablyCarl committed:
Merge branch 'dev' of https://github.com/ProbablyCarl/BoH-Bay-1 into dev
